File: src/product-service/models/order_result_item.go
package models

import uuid "github.com/satori/go.uuid"

type OrderResultItem struct {
	ID            int         ` json:"order_result_item_id" gorm:"primaryKey"`
	ProductID     int         ` json:"product_id"`
	Product       Product     `json:"product"`
	ProductUUID   uuid.UUID   `json:"product_uuid"`
	WarehouseID   int         ` json:"warehouse_id"`
	Warehouse     Warehouse   `json:"warehouse"`
	OrderResultID int         ` json:"order_result_id"`
	OrderResult   OrderResult `json:"order_result"`
}

File: src/ProductManagement/ProductManagement.Models/OrderResultItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ProductManagement.Models
{
    public class OrderResultItem
    {
        public long Id { get; set; }
        public long ProductId { get; set; }
        public Product Product { get; set; }
        public string ProductUUID { get; set; }
        public long WarehouseId { get; set; }
        public Warehouse Warehouse { get; set; }
        public long OrderResultId { get; set; }
        public OrderResult OrderResult { get; set; }
    }
}
